styles/main.css

```diff
@@ -111,7 +111,7 @@ footer hr {
     background-size: contain;
     position: absolute;
     top: -10px;
-    right: 7px;
+    right: -7px;

```

Comment:
Use negative numbers on right-positioned elements to move them to the right (as requested by the lab) instead of to the left